pkg/test/framework/components/istio/config.go

```diff
@@ -149,6 +149,12 @@ type Config struct {
 
 	// Indicates that the test should deploy Istio's using helm charts
 	DeployHelm bool
+
+	// Version of the Istio control plane to deploy for this instance
+	Version string
+
+	// Revision for this control plane installation
```

Comment:
What's the difference between version and revision? IIUC you're using `version` to represent the actual Istio control plane version being installed and `revision` to be the label used. Could we just infer `revision` rather than making it configurable?

That would also mean that we'd want to get rid of any tests that are manually specifying revision (I think there's only one, maybe?), but I think that's fine ... upgrade tests will replace these anyway.

Reply:
Made it so that we infer `revision` from `version` like you mentioned. Existing tests should keep working I think? `revision` here doesn't actually change the revision, it just makes it so that we can ask an `istio` instance which revision it's targeting (revisions are included in pre-generated manfiests)